### PR TITLE
Added DateLocalization for Danish

### PR DESCRIPTION
**Pull request by mrSkovgaard**
As suggested in @albertospelta 's answer to #563 I have added Danish translation for date table columns. I have translated column names in all DateLocalization files. Br, Niels

### CODE REVIEW

File: src/Assets/ManageDates/Templates/DateLocalization-01.json [hunk 1]
```diff
@@ -1109,6 +1109,191 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-01.json:1114]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-02.json [hunk 2]
```diff
@@ -1097,6 +1097,185 @@
           "DisplayFolders": ""
         }
       ]
-    }
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-02.json:1102]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-03.json [hunk 3]
```diff
@@ -1162,6 +1162,191 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-03.json:1167]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-04.json [hunk 4]
```diff
@@ -1174,6 +1174,191 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-04.json:1179]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-05.json [hunk 5]
```diff
@@ -1066,6 +1066,174 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-05.json:1071]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-06.json [hunk 6]
```diff
@@ -1001,6 +1001,174 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-06.json:1006]:
```suggestion
      "Iso": "da",
```

File: src/Assets/ManageDates/Templates/DateLocalization-07.json [hunk 7]
```diff
@@ -1565,6 +1565,267 @@
           "DisplayFolders": ""
         }
       ]
+    },
+    {
+      "Iso": "dk",
```
Comment (albertospelta) [inline src/Assets/ManageDates/Templates/DateLocalization-07.json:1570]:
```suggestion
      "Iso": "da",
```

**Review by albertospelta — CHANGES_REQUESTED** [review]
Thanks for the PR! Would you mind applying the requested changes? 
It's just a small fix to the ISO language code.

**Review by albertospelta — APPROVED** [review]

**Comment by mrSkovgaard** [comment]
Suggestions committed. Merry Christmas :-)

**Comment by albertospelta** [comment]
Added in version [1.0.2](https://github.com/sql-bi/Bravo/releases/tag/v1.0.2)

**Pull request merged**